SocialFilter/sections/general_extensions.txt

```diff
@@ -232,7 +232,7 @@ crimea-kurort.com#$#img[width="88"][height="31"] {visibility: hidden!important;}
 maskme.ru#$#img[width="88"][height="31"] {visibility: hidden!important;}
 !
 gamesradar.com#$#.shareSocialArticle-wrap { visibility: hidden; }
-softwares-serials.in#$##FTslide { visibility: hidden;!important; }
+softwares-serials.in#$##FTslide { visibility: hidden!important; }

```

Comment:
We also add space after the value.
```
softwares-serials.in#$##FTslide { visibility: hidden !important; }
```

Btw, is that domain alive?

Reply:
@zloyden As I see, no :sweat_smile: So I remove the rule instead

Reply:
I'm just testing the parser

Reply:
Done